### PR TITLE
Remove sanctuary

### LINKED ISSUE #19: Remove sanctuary
It looks like sanctuary is the vast majority of the bundle size. I don't think it's heavily used by this library so we should replace it IMO. Both index.js in the image are from sanctuary:

screenshot from 2017-10-11 11-46-14

### PR DESCRIPTION
**Pull request by noisecapella**
Fixes #19

### CODE REVIEW

File: src/django_csrf_fetch.js [hunk 1]
```diff
@@ -87,22 +76,17 @@ const handleEmptyJSON = json => (
  *  - response JSON is returned in place of response
  */
 export const fetchJSONWithCSRF = async (input: string, init: Object = {}): Promise<*> => {
-  let response = await fetch(input, formatJSONRequest(init))
-  let text = await response.text()
+  const response = await fetch(input, formatJSONRequest(init))
+  const text = await response.text()
+
+  const jsonFriendlyText = handleEmptyJSON(text)
+  const json = JSON.parse(jsonFriendlyText)
```
Comment (aliceriot) [inline src/django_csrf_fetch.js]: Can we just use `response.json()` here instead?

Comment (noisecapella) [reply src/django_csrf_fetch.js]: I think we wanted to handle empty responses with a `{}`, but I can't remember why we wanted to do that. I'm fine with switching to `.json()` if you are

Comment (aliceriot) [reply src/django_csrf_fetch.js]: I think it's probably the cleaner approach. If it's an empty response, do you know what's returned by `.json()`?

Comment (noisecapella) [reply src/django_csrf_fetch.js]: I tested it and it converts it to an empty string, which I think makes sense

**Comment by noisecapella** [comment]
Should I increment the version in this PR too?

File: src/django_csrf_fetch.js (hunk 1 above)
Comment (aliceriot) [reply src/django_csrf_fetch.js]: yeah, the only issue I see with that is that we'd be calling `...""`, which in the context of an object literal wouldn't make much sense. So we maybe want to do something:

```js
let json = await response.json()
json = json === ""
  ? {}
  : json
```

or something along those lines, to ensure we're always going to be `...`-ing an object

**Comment by aliceriot** [comment]
yea we should increment the version so we can push an update to npm

**Comment by noisecapella** [comment]
Comments addressed

**Pull request merged**